tests/js/app/interfaces/testCard.js

```diff
@@ -141,9 +141,8 @@ describe('Card interface', function () {
 
     it('sets a thumbnail frame visible if model has a thumbnail uri', function () {
         let engine = MockEngine.mock_default();
-        engine._lookup_ekn_uri.and.callFake(() => {
-            return Gio.Vfs.get_local().get_file_for_path('/dev/zero');
-        });
+        let zeroes = Gio.Vfs.get_local().get_file_for_path('/dev/zero');
```

Comment:
I wonder if we even need to single out vfs local for this. Shouldn't `Gio.File.new_for_uri('file:///dev/zero')` work?